releasing/plugin-commands-publishing/src/publish.ts

```diff
@@ -137,6 +138,27 @@ export async function publish (
   } & Pick<Config, 'allProjects' | 'gitChecks' | 'ignoreScripts' | 'publishBranch' | 'embedReadme' | 'packGzipLevel'>,
   params: string[]
 ) {
+  const tokenHelpers = Object.entries(opts.rawConfig).filter(([key]) => key.endsWith(':tokenHelper'))
+  const tokenHelpersFromArgs = opts.argv.original.filter(arg => arg.includes(':tokenHelper=')).map(arg => [arg.split('=')[0], arg.split('=')[1]] as [string, string])
+
+  const parsedTokenHelpers = tokenHelpers.concat(tokenHelpersFromArgs).map(([key, value]) => {
+    // Extracting the registry and port from the npm configuration key
+    const registryMatch = key.match(/\/\/(.*?):(\d+)\/:/)
+    if (!registryMatch) {
+      throw new Error(`Invalid registry format in key: ${key}`)
+    }
+
+    const [, registryHost, registryPort] = registryMatch
+    const registry = registryPort ? `//${registryHost}:${registryPort}/` : `//${registryHost}/`
+
+    const parsedToken = loadToken(value, key)
```

Comment:
The token helper can return either `Bearer TOKEN`, which should be passed to `_authToken` (only the TOKEN part), or `Basic TOKEN`, which should be passed to `_auth`.